charts/minio-manager/templates/minio-manager-global-job.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: minio-manager-global
```

Comment:
If you change the policies, a new job will not start to apply them. If I remember, you can add a hash of the values to start a new job

Reply:
@vQuagliaro  good point, I already forgot the immutability of jobs.
To solve this, I'm using the [helm hooks](https://helm.sh/docs/topics/charts_hooks/) which can delete jobs before re installing/updating the chart.
I have tested locally with helm install and helm upgrade, which runs well the job on a policy change.

From the [argocd docs](https://argoproj.github.io/argo-cd/user-guide/helm/#helm-hooks), it should also work, but not tested locally.